src/com/hms_networks/americas/sc/taginfo/TagInfoEnumeratedIntToString.java

```diff
@@ -0,0 +1,88 @@
+package com.hms_networks.americas.sc.taginfo;
+
+import java.util.ArrayList;
+
+/**
+ * Class to hold information about a tag and its configuration when the tag is an {@code int} to
+ * {@link String} enumeration tag.

```

Comment:
Should its be "IntToStringEnumeration" instead of  "{@code int} to {@link String} enumeration"?

Reply:
updated to better reflect that this aligns with TagType#INTEGER_MAPPED_STRING